introspection-engine/introspection-engine-tests/tests/re_introspection/relation_mode/mssql.rs

```diff
@@ -0,0 +1,402 @@
+// @@map
+mod at_at_map {
+    use indoc::indoc;
+    use introspection_engine_tests::test_api::*;
+
+    // referentialIntegrity = "prisma" with @@map loses track of the relation policy ("prisma") but preserves relations.
```

Comment:
I think this isn't accurate anymore with the other PR being merged, right?

Reply:
Yeah I was waiting to merge the other PR before updating the tests.